File: calc/src/main/scala/com/pharbers/aqll/alcalc/alCommon/alFromJson.java
package com.pharbers.aqll.alcalc.alCommon;

import com.google.gson.Gson;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by qianpeng on 2017/4/13.
 */
public class alFromJson {
	public static Map<String, String> formJson(String resp) {
		Gson gson = new Gson();
		return  gson.fromJson(resp, Map.class);
	}
}

File: calc/src/main/scala/com/pharbers/aqll/alcalc/alCommon/alFromJson.scala
package com.pharbers.aqll.alcalc.alCommon

import com.google.gson.Gson

/**
  * Created by qianpeng on 2017/5/16.
  */
object alFromJson {
	def apply(resp: String) = formJson(resp)
	def formJson(resp: String) = new Gson().fromJson(resp, classOf[Map[String, String]])
}
